### ISSUE TITLE
can't select calendars created from task lists

### ISSUE THREAD

**Issue by bgravato** [nextcloud/calendar#2837]
I'm running Nextcloud 20, Calendar 2.1.3 and Tasks 0.13.6.

All my task lists show up on the left side menu of Calendar as existing calendars (as it should I believe), but when I create a new event or modify an existing one, I can't assign it to any of the calendars created from tasks lists except Personal calendar/task list (maybe that one was created from within calendar? not sure).

If I create a new calendar (either as calendar only or both calendar and task list) it will show both on the left side calendar list and on the event's drop down menu for selecting the calendar and I can assign events to it. If I created it with the task list option it will also show in Tasks lists as expected.

If I create a new task list in Tasks, it will show on the left side menu of the calendar (and I can even delete it from there and it will vanish from both apps), but it will not show on the event's drop down menu, so I can't assign any event to it.

If I create a calendar with the same name as the task list I'll just end up with two items on the left side menu with the same name...

Not sure if this is a bug in Tasks or Calendar (or both, or none...), but doesn't look like an expected behaviour...

Anyone having the same issue or able to reproduce it? Any fix?

Only workaround I see is recreating all lists from within the Calendar app and move all tasks to there... but I already have quite a few tasks and task lists created, so this would be very time consuming...

Sorry if it's already reported, I couldn't find anything related, but it's not easy to search about this with any really unique keywords, is it?

**Comment by raimund-schluessler** [comment]
Lists created from within the Tasks app should not show up as calendars, this is the standard behavior. We had multiple reports were users were not aware that a "list" (in Tasks) is also a "calendar" (in Calendar) and they deleted the task list, hence losing all their events.

That's why we decided to create lists in Tasks as tasks-only and don't allow to store events in such lists. There is no standard-conform way to change what a list/calendar can store after it's creation, sorry. Your only way is to export the tasks, create a calendar+list in the Calendar app and reimport all tasks with the Calendar app.

**Comment by bgravato** [comment]
Thank you for your quick reply.

> Lists created from within the Tasks app should not show up as calendars, this is the standard behavior. 

This wasn't what I was watching, but I figured out why...

This is what was happening:
* when I create a new task list in Tasks, it will immediately display in Calendar's left side pane.
* exact same thing happens when I create a new board in Deck.

After closer inspection to Calendar's settings I noticed that the option "Show tasks in calendar" was enabled. If I uncheck this option all task lists (that aren't also calendars) and deck boards disappear from Calendar's left side pane.

I see this feature can be of interest, but to avoid confusion, I'd like to suggest that task lists that are not calendars, show up differently in the calendar list. For example, this already (sort of) happens with Deck boards, their name is prefixed with "Deck: ".

This does not happen for task lists... So the most obvious solution would be to prefix their names with "Task: " or "Tasks: ".

Other alternatives could be putting the name in italics, greyed out, etc or grouped under some label like "Task lists" or "Deck boards".

Kindly,
Bruno
